tests/qos/test_qos_sai.py

```diff
@@ -138,8 +138,6 @@ class TestQosSai(QosSaiBase):
         'Arista-7050CX3-32S-D48C8'
     ]
 
-    BREAKOUT_SKUS = ['Arista-7050-QX-32S']
```

Comment:
why is this removed?

Reply:
move this to qos_sai_base.py

because in dutConfig() method:
we only remove low speed port for "BREAKOUT_SKUS" for safe.

            if src_dut.facts['hwsku'] in **self.BREAKOUT_SKUS** and 'backend' not in topo:
                for speed, portlist in port_speeds.items():
                    if int(speed) < 40000:
                        for portname in portlist:
                            low_speed_portIds.append(src_mgFacts["minigraph_ptf_indices"][portname])